databricks_cli/groups/cli.py

```diff
@@ -0,0 +1,181 @@
+"""Provide the API methods for GROUPs Databricks REST endpoint."""
+# Databricks CLI
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+
+from databricks_cli.groups.api import GroupsApi
+from databricks_cli.utils import eat_exceptions, CONTEXT_SETTINGS, pretty_format
+from databricks_cli.configure.config import provide_api_client, profile_option
+from databricks_cli.version import print_version_callback, version
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Add an existing group to another existing group.')
+@click.option('--group', '-g', required=True,
+              help="The name of the group to be added to the parent group.")
+@click.option('--parent', '-p', required=True,
+              help="The name of the parent group.")
+@profile_option
+@eat_exceptions
+@provide_api_client
+def add_group_to_parent_group_cli(api_client, group, parent):
```

Comment:
Ideally this interface should match identically to the API https://docs.databricks.com/api/latest/groups.html#add-member.

Can we make the command called `add-member` taking three options? `--user-name`, `--parent-name`, `--group-name`?

Reply:
Also, let's remove the single character options please.